modules/user/main.tf

```diff
@@ -0,0 +1,9 @@
+
+variable "username" {
```

Comment:
Great point, I should've elaborated on this more. A few reasons:

* I envision the user module as taking a `membership_level` parameter in the near future, with possible values of `"collaborator"`, `"member"`, or `"owner"`, the latter two of which will result in `github_membership` resources.
* The Terraform labels are currently based on the usernames, but there's not reason we have to stick with that. We might find it easier to use collaborator's firms and real names (when they are comfortable with that) rather than GH usernames -- for example, instead of `module.user_kdmccormick`, I'd be `module.user_tcril_kyle_mccormick`.
* It makes it marginally easier to update the code to reflect username changes.
 
That being said, this might be over-engineering, and I wouldn't be against switching to simple usernames after the repo transition if other feel that way.